<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptEditor</class>
 <widget class="QWidget" name="ScriptEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script Editor</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>731</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>721</width>
       <height>361</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>706</width>
        <height>359</height>
       </rect>
      </property>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>721</width>
         <height>371</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="New">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>90</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Open">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>90</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>90</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Compile">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>0</y>
     <width>90</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Compile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>0</y>
     <width>90</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
